SecuritySystemUWP/SecuritySystemUWP/Storage/OneDrive.cs

```diff
@@ -109,7 +105,7 @@ public DateTime LastUploadTime
                     //Delete all pictures from the day
                     foreach (string picture in pictures)
                     {
-                        await oneDriveConnector.DeleteFileAsync(picture, folder);
+                        await deleteWithRetry(picture, folder, 1);
```

Comment:
Possibly. Interestingly, there wasn't one originally. Additionally, for the equivalent area in the "upload picture" section, there are two catches, instead of the one currently in this method.

I feel like it will be useful to have different telemetry for a failure on a single file vs. a failure in the setup operations for these methods. I'll make some changes, let me know what you think after my next push.